### PR TITLE
Feature/fbt 164 approver care packages

### PR DESCRIPTION
**Pull request by jamesoutterside**

### CODE REVIEW

File: components/kanban/__tests__/__snapshots__/cards.test.tsx.snap [hunk 1]
```diff
@@ -41,8 +41,8 @@ exports[`KanbanCard card is assigned to user, assignment tag is present and high
       </span>
     </div>
     <div
-      class="myCircle"
-      title="Assigned to Foo Tester (you)"
+      class="assignmentCircle"
```
Comment (OmegaDroid) [inline components/kanban/__tests__/__snapshots__/cards.test.tsx.snap:44]: I don't think this should have changed

File: components/kanban/approver.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      <SplitPanes>
+        <CollapsibleSideBarPane>
+          <ApproverFilters cap={"£20k"}/> {/* TODO Cap will need to be driven from the API/what cols a user can see */}
+        </CollapsibleSideBarPane>
+        <MainPane>
+          <KanbanColumnsWrapper>
+            <KanbanColumn filter={amountFilter20k} label={`Up to £20k`}/>
+            <KanbanColumn filter={amountFilter100k} label={`Up to £100k`} />
+            <KanbanColumn filter={amountFilter500k} label={`Up to £500k`} />
+            <KanbanColumn filter={amountFilter1m} label={`Up to £1m`} />
```
Comment (OmegaDroid) [inline components/kanban/approver.tsx:64]: Based on the filter this label should probably be greater than 500k, may not be an issue though based on the actual values of care packges

Comment (jamesoutterside) [reply components/kanban/approver.tsx:64]: ah yeah makes sense, will limit it.

File: components/kanban/__tests__/__snapshots__/cards.test.tsx.snap (hunk 1 above)
Comment (jamesoutterside) [reply components/kanban/__tests__/__snapshots__/cards.test.tsx.snap:44]: Interesting assuming that is caused by dropping `@/context/session` will take a look.

**Review by OmegaDroid — APPROVED** [review]

**Pull request merged**